openmetadata-ui/src/main/resources/ui/src/pages/ServicesPage/ServicesPage.test.tsx

```diff
@@ -0,0 +1,132 @@
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { userPermissions } from '../../utils/PermissionsUtils';
+import ServicesPage from './ServicesPage';
+
+jest.mock(
+  '../../components/Settings/Services/Ingestion/IngestionPipelineList/IngestionPipelineList.component',
+  () => ({
+    IngestionPipelineList: jest
+      .fn()
+      .mockReturnValue(<>mockIngestionPipelineList</>),
+  })
+);
+
+jest.mock('../../components/Settings/Services/Services', () => {
+  return jest.fn().mockReturnValue(<>mockServices</>);
+});
+
+jest.mock('../../components/PageLayoutV1/PageLayoutV1', () => {
+  return jest.fn().mockImplementation(({ children }) => <>{children}</>);
+});
+
+jest.mock(
+  '../../components/common/ErrorWithPlaceholder/ErrorPlaceHolder',
+  () => {
+    return jest.fn().mockReturnValue(<>mockErrorPlaceHolder</>);
+  }
+);
+
+jest.mock(
+  '../../components/common/TitleBreadcrumb/TitleBreadcrumb.component',
+  () => {
+    return jest.fn().mockReturnValue(<>mockTitleBreadcrumb</>);
+  }
+);
+
+const mockEntityPermissions = {
+  Create: true,
+  Delete: true,
+  ViewAll: true,
+  ViewBasic: true,
+  EditAll: true,
+  EditTags: true,
+  EditDescription: true,
+  EditDisplayName: true,
+  EditCustomFields: true,
+};
+
+jest.mock('../../context/PermissionProvider/PermissionProvider', () => ({
+  usePermissionProvider: jest
+    .fn()
+    .mockImplementation(() => ({ permissions: mockEntityPermissions })),
+}));
+
+jest.mock('../../hooks/authHooks', () => ({
+  useAuth: jest.fn().mockReturnValue({
+    isAdminUser: true,
+  }),
+}));
+
+jest.mock('../../utils/GlobalSettingsUtils', () => ({
+  getSettingPageEntityBreadCrumb: jest.fn().mockReturnValue({
+    getSettingPageEntityBreadCrumb: jest.fn().mockReturnValue([]),
+  }),
+}));
+
+// jest.mock('../../utils/ServiceUtils', () => ({
+//   getResourceEntityFromServiceCategory: jest.fn().mockReturnValue(null),
+// }));
+
+jest.mock('../../utils/PermissionsUtils', () => ({
+  userPermissions: {
+    hasViewPermissions: jest.fn().mockReturnValue(true),
+  },
+}));
+
+describe('ServicesPage', () => {
+  it('should render Services tab', async () => {
+    await act(async () => {
+      render(
+        <MemoryRouter initialEntries={['/services/databases']}>
+          <Route component={ServicesPage} path="/services/:tab" />
+        </MemoryRouter>
+      );
+    });
+
+    expect(await screen.findByText('Services')).toBeInTheDocument();
```

Comment:
@chirag-madlani , I believe this will check for text we have as tab label, can we add assertion for checking if services tab is active then list of services should render.